content/en/user-guide/aws/elastic-kubernetes-service/index.md

```diff
@@ -1,82 +1,119 @@
 
-There are two modes for creating EKS clusters on LocalStack:
-* spinning up an embedded kube cluster in your local Docker engine (preferred, simpler), or
-* using an existing Kubernetes installation you can access from your local machine (defined in `$HOME/.kube/config`)
+## Introduction
 
-## Auto-installing an embedded Kubernetes cluster
+Elastic Kubernetes Service (EKS) is a managed Kubernetes service that makes it easy to run Kubernetes on AWS without installing, operating, and maintaining your own Kubernetes control plane or worker nodes. Kubernetes is an open-source system for automating containerised applications' deployment, scaling, and management.
 
-The default method for creating Kubernetes clusters via the local EKS API is to spin up an embedded [k3d](https://k3d.io) kube cluster within Docker. LocalStack handles the download and installation transparently - on most systems the installation is performed automatically, and no customizations should be required.
+LocalStack supports Elastic Kubernetes Service via the Pro/Team offering, allowing you to use the EKS APIs in your local environment to spin up embedded Kubernetes clusters in your local Docker engine or use an existing Kubernetes installation you can access from your local machine (defined in `$HOME/.kube/config`). The supported APIs are available on our [API coverage page](https://docs.localstack.cloud/references/coverage/coverage_eks/), which provides information on the extent of EKS's integration with LocalStack.
+
+## Getting started
+
+This guide is designed for users new to Elastic Kubernetes Service and assumes basic knowledge of the AWS CLI and our [`awslocal`](https://github.com/localstack/awscli-local) wrapper script. To interact with the Kubernetes cluster, you should also install [`kubectl`](https://kubernetes.io/docs/tasks/tools/).
+
+Start your LocalStack container using your preferred method. We will demonstrate how you can auto-install an embedded Kubernetes cluster, configure ingress, and deploy a sample service with ECR.
+
+### Create an embedded Kubernetes cluster
+
+The default approach to create Kubernetes clusters using the local EKS API is by setting up an embedded [k3d](https://k3d.io/) kube cluster within Docker. LocalStack seamlessly manages the download and installation process, making it hassle-free for users. In most cases, the installation is automatic, eliminating the need for any manual customizations.
 
 A new cluster can be created using the following command:
+
+You can create a new cluster using the [`CreateCluster`](https://docs.aws.amazon.com/eks/latest/APIReference/API_CreateCluster.html) API. Run the following command:
+
 {{< command >}}
-$ awslocal eks create-cluster --name cluster1 --role-arn arn:aws:iam::000000000000:role/eks-role --resources-vpc-config '{}'
+$ awslocal eks create-cluster \
+  --name fancier-cluster \
```

Comment:
nit: "fancier-*" seems a bit unexpected here. It makes sense, when looking at the `fancier-nginx` pod further below, but wondering if we should err on the "unfancy" side here :) and use typical test names like "cluster1", "pod1, etc.. (not critical at all, though - feel free to merge as-is as well 👍 )